examples/index.html

```diff
@@ -1,19 +1,21 @@
+<base target="example">
```

Comment:
Adding the base here allows to simplify all of the individual targets. I can of course revert this if needed @Kalabasa